### PR TITLE
Implemented normalisation for weights with sum larger than one. Also …

### PR DESCRIPTION
**Pull request by BStoelzner**
This fixes the normalisation issue in #146

### CODE REVIEW

**Review by aimalz — CHANGES_REQUESTED** [review]
This is good! It just needs one little unit test to be complete.

**Comment by BStoelzner** [comment]
@aimalz I've added a unit test, but forgot to let you know. Feel free to take a look :)

**Comment by BStoelzner** [comment]
The conflict should be resolved now.

File: tests/qp/test_ensemble.py
[the remark this answers is not in the document]
Comment (aimalz) [reply tests/qp/test_ensemble.py:1]: Ack, sorry, I meant this comment as a suggestion that could be made now or in the future but failed to change the status of the review to approved to actually indicate that, my bad!

**Review by aimalz — APPROVED** [review]
Note the to-do item for an additional test to catch the original way the bug cropped up. I can write it later this week and add it to this PR, or we can make an issue to come back to it some other time.

**Pull request merged**